plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -136,6 +136,26 @@ public void testFlushHiveMetastoreCacheProcedureCallable()
         queryRunner.execute(renamedColumnQuery);
     }
 
+    @Test
```

Comment:
TestCachingHiveMetastoreWithQueryRunner -- this is an odd name for a test class.

It's named like this to differentiate from a unit test TestCachingHiveMetastore, and so it should be called TestCachingHiveMetastoreQueries (alas, this already exists! let's merge the classes -- as a follow-up)

Reply:
Yes we could merge them there are some differences in QueryRunner setup, but i think we can manage to adjust.